test/trace_processor/diff_tests/parser/power/tests_power_rails.py

```diff
@@ -151,6 +151,93 @@ def test_power_rails_well_known_power_rails(self):
         "power.rails.gpu",999.000000,1
         """))
 
+  def test_power_rails_session_uuid(self):
+    return DiffTestBlueprint(
+        trace=TextProto(r"""
+        packet {
+          power_rails {
+            session_uuid: 100
+            rail_descriptor {
+              index: 0
+              rail_name: "CPU_RAIL"
+              subsys_name: "cpu"
+              sampling_rate: 1000
+            }
+          }
+        }
+        packet {
+          power_rails {
+            session_uuid: 200
+            rail_descriptor {
+              index: 0
+              rail_name: "GPU_RAIL"
+              subsys_name: "gpu"
+              sampling_rate: 2000
+            }
+          }
```

Comment:
We should also test for a situation where two dfiferent sessions at the same rail index have the same rail name and verify there's no cross talk. We can use `extract_arg(dimension_arg_set_id, 'session_uuid')` in SQL for that.